tests/integration/components/bs-tooltip-test.js

```diff
@@ -9,6 +9,7 @@ import {
   tooltipArrowClass,
   tooltipPositionClass,
   visibilityClass,
+  versionDependant,
```

Comment:
Small typo causing it to fail:

```suggestion
  versionDependent,
```